index.html

```diff
@@ -422,6 +423,12 @@
         </paper-toolbar>
 
         <iron-pages id="pages" selected="[[tab]]" attr-for-selected='id'>
+
+          <template is="dom-if" if="[[showError(tab)]]">
+            Page '{{tab}}' not found.
```

Comment:
@juliocri I think since `this.tab` doesn't match any element id in the iron-page children, iron-pages won't show the `<error-page>` element (it is looking for the element with the `id=[[tab]]`. The text shows up because it isn't wrapped in a tag, so isn't being managed by the `iron-pages` selection.

Have you tried setting `fallback-selection="error"` on the `iron-pages` tag? Then if no child of the iron-pages exists that has an id that matches `tab`, iron-pages should show the fallback-selection. I'm not sure if that will, in turn, set `tab="error"` which would break the string `Page {{tab}} not found.` from being able to show the page that is missing.